samples/Common/cs/SkillHelper.cs

```diff
@@ -90,21 +151,67 @@ public static string GetSkillFeatureDescriptorString(ISkillFeatureDescriptor des
         /// </summary>
         /// <param name="desc"></param>
         /// <returns></returns>
-        public static List<KeyValuePair<string, string>> GetSkillInformationStrings(ISkillDescriptor desc)
+        public static List<KeyValuePair<string, string>> GetSkillInformationStrings(SkillInformation info)
```

Comment:
Update ConceptTaggerSample_UWP\MainPage.xaml.cs line 93 to match

Reply:
thx, note that I plan on coalescing the other samples to the new display formats later on